cmd/systray/systray.go

```diff
@@ -375,6 +402,230 @@ func sendNotification(title, content string) {
 	}
 }
 
+func (menu *Menu) rebuildExitNodeMenu(ctx context.Context) {
+	status := menu.status
+	menu.exitNodes = systray.AddMenuItem("Exit Nodes", "")
+	time.Sleep(newMenuDelay)
+
+	// register a click handler for a menu item to set nodeID as the exit node.
+	onClick := func(item *systray.MenuItem, nodeID tailcfg.StableNodeID) {
+		go func() {
+			for {
+				select {
+				case <-ctx.Done():
+					return
+				case <-item.ClickedCh:
+					select {
+					case <-ctx.Done():
+						return
+					case menu.exitNodeCh <- nodeID:
+					}
+				}
+			}
+		}()
+	}
+
+	noExitNodeMenu := menu.exitNodes.AddSubMenuItemCheckbox("None", "", status.ExitNodeStatus == nil)
+	onClick(noExitNodeMenu, "")
+
+	// Show recommended exit node if available.
+	if status.Self.CapMap.Contains(tailcfg.NodeAttrSuggestExitNodeUI) {
+		sugg, err := localClient.SuggestExitNode(ctx)
+		if err == nil {
```

Comment:
```go
if err != nil {
   // Ignore.
   continue
}
// outdented code
```

Reply:
oh, it's not in a loop, nevermind.

Reply:
this isn't in a loop, so can't continue

Reply:
I've got some pretty deeply nested loops later on though that could probably be simplified by continuing early.